high-throughput.env

```diff
@@ -0,0 +1,14 @@
+# SPDX-FileCopyrightText: 2021-2024 Brighter AI Technologies GmbH
+# SPDX-License-Identifier: MIT
+REDACT_GPU_IMAGE="docker.brighter.ai/redact-gpu:3.3.0-A10"
+
+NUM_PROC_IMAGE_COMPOSER=10
+NUM_PROC_DETECTION=10
+NUM_PROC_FACE_TRACKING=10
+NUM_PROC_LICENSE_PLATE_TRACKING=10
+NUM_PROC_FACE_LANDMARKS_SMOOTHING=2
+NUM_PROC_LICENSE_PLATE_LANDMARKS_SMOOTHING=2
+NUM_PROC_SPLITTER=50
+NUM_PROC_AGGREGATOR=50
+NUM_PARALLEL_ACTIVE_VIDEOS=50
+SPLITTER_MAX_ACTIVE_FRAMES=1500
```

Comment:
Sorry, here I have a new version after testing with DNAT more thoroughly:
      NUM_PROC_IMAGE_COMPOSER: 15
      NUM_PROC_DETECTION: 10
      NUM_PROC_FACE_TRACKING: 15
      NUM_PROC_LICENSE_PLATE_TRACKING: 15
      NUM_PROC_FACE_LANDMARKS_SMOOTHING: 5
      NUM_PROC_LICENSE_PLATE_LANDMARKS_SMOOTHING: 5
      NUM_PROC_FACE_GENERATOR: 35
      NUM_PROC_SPLITTER: 50
      NUM_PROC_AGGREGATOR: 50
      NUM_PARALLEL_ACTIVE_VIDEOS: 50
      SPLITTER_MAX_ACTIVE_FRAMES: 200

Reply:
I need to correct something after more input from core: 
NUM_PARALLEL_ACTIVE_VIDEOS: 1 (not 50)